package/MDAnalysis/analysis/helix_analysis.py

```diff
@@ -387,11 +388,13 @@ class HELANAL(AnalysisBase):
         'local_screw_angles': (-2,),
     }
 
-    def __init__(self, universe, select='name CA', ref_axis=[0, 0, 1],
+    def __init__(self, universe, select='name CA', ref_axis=None,
```

Comment:
What about using a tuple instead of a list?

Reply:
Sure I can do that

Reply:
I meant using it as an argument (instead of adding the logic with `None`). Because tuples are immutable (while lists are mutable).

Reply:
> using

Oh that makes sense